tests/integration/run-tests.sh

```diff
@@ -234,6 +234,9 @@ run-single-test() {
 
 mecho blue "Building pirouette"
 cabal build
+mecho blue "Run unit tests"
```

Comment:
Maybe do something like:

```
if $onci; then
    cabal test
fi
```

This way we don't run the cabal tests every single time we call `run-tests.sh`